Articles/Blog/2023-10-fear-of-large-codes.md

```diff
@@ -45,35 +44,48 @@ It is natural to ask at this point: why is this fear of large codes so common?
 After all, not all students are like the one mentioned at the beginning of this article, praised for their "lack of fear" of getting their hands dirty with large codes.
 What if there was a way for *most* students to work on large codes without necessarily needing years of development experience and, most importantly, without being afraid of them?
 You may think this a utopian view, but I would argue that such a reality *could* exist.
-Imagine a code (started as a small project, but on its way to becoming a large one) developed from the beginning with *accessibility* and *sustainability* in mind: the design is modular and flexible (e.g., the open-closed principle is meticulously met), documentation is complete, and the test suite is well-developed.
+Imagine a code (started as a small project, but on its way to becoming a large one) developed from the beginning with *accessibility* and *sustainability* in mind: the design is modular and flexible (e.g., in object-oriented programming (OOP) the [SOLID principles](https://www.youtube.com/watch?v=Ntraj80qN2k) are meticulously met), documentation is complete, and the test suite is well-developed.
```

Comment:
@bartlettroscoe I think it would be great to turn these recommendations into a CC that we could refer to in Francesco's blog.  Would you be willing and able to whip something up in time for a 26 Oct publication date?

Reply:
> @bartlettroscoe I think it would be great to turn these recommendations into a CC that we could refer to in Francesco's blog. Would you be willing and able to whip something up in time for a 26 Oct publication date?

@bernhold, you mean a CC article for the SOLID principles?